test/metabase/query_processor/middleware/cache_test.clj

```diff
@@ -312,6 +312,11 @@
               _               (while (a/poll! save-chan))
               _               (mt/wait-for-result save-chan)
               cached-result (qp/process-query query)]
+          (is (=? {:cache/details {:stored true
+                                   :hash some?}
+                   :row_count 1
+                   :status :completed}
```

Comment:
Sorry but I've come to be very pedantic about aligning map values 😄

Reply:
it's a test written by somebody else and I just did not want to introduce the churn :)

Reply:
thanks for formatting the whole function, but i'd have been happy with just your new lines of code :-)